rcl/include/rcl/logging_rosout.h

```diff
@@ -31,26 +31,6 @@ extern "C"
 {
 #endif
 
-/// The default qos profile setting for topic /rosout
-/**
- * - depth = 1000
- * - durability = RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL
- * - lifespan = {10, 0}
- */
-RCUTILS_DEPRECATED_WITH_MSG("use rmw_qos_profile_rosout_default instead")
-static const rmw_qos_profile_t rcl_qos_profile_rosout_default =
```

Comment:
this has been deprecated in kilted, this PR removes this for next L-turtle.

Reply:
`rcl_qos_profile_rosout_default` is mentioned in the documentation here: https://github.com/ros2/rcl/blob/6209445e364cbd90e7f81782fedda7da9e19e1ba/rcl/include/rcl/node_options.h#L67

it should be updated to `rmw_qos_profile_rosout_default` like the code

Reply:
@christophebedard good catch, i will update it! thanks!